solutions/04-getting-started-js/sum_concat.js

```diff
@@ -0,0 +1,14 @@
+function sum (a, b) {
+  if (typeof a === 'number' && typeof b === 'number') {
+    return a + b
+  } else throw new TypeError('Error: arguments of sum should be numbers.')
```

Comment:
When using if/else statements, it's always a good practice to wrap the blocks after them, even if you don't need to. Ex:
```javascript
if (condition) {
 // do smt if condition is true
} else {
 // do smt if condition is false
}
```
It's more readable -- _remember, we write code for humans, not machines!_ -- and lowers the chance that a developer will oversee something. It's better to learn it now, while still in the beginning of getting started with the language :)